Package.swift

```diff
@@ -68,49 +69,49 @@ let package = Package(
     cxxLanguageStandard: .cxx17
 )
 
-// Add the ORT iOS Pod archive as a binary target.
+// Add the ORT CocoaPods C/C++ pod archive as a binary target.
 //
 // There are 2 scenarios:
+// - Target will be set to a released pod archive and its checksum.
 //
-// Release version of ORT SPM github repo:
-//    Target will be set to the latest released ORT iOS pod archive and its checksum.
-//
-// Current main of ORT SPM github repo:
-//    Target will be set to the pod archive built in sync with the current main objective-c source code.
+// - Target will be set to a local pod archive.
+//   This can be used to test with the latest (not yet released) ORT Objective-C source code.
 
-// CI or local testing where you have built/obtained the iOS Pod archive matching the current source code.
-// Requires the ORT_IOS_POD_LOCAL_PATH environment variable to be set to specify the location of the pod.
-if let pod_archive_path = ProcessInfo.processInfo.environment["ORT_IOS_POD_LOCAL_PATH"] {
-    // ORT_IOS_POD_LOCAL_PATH MUST be a path that is relative to Package.swift.
+// CI or local testing where you have built/obtained the pod archive matching the current source code.
+// Requires the ORT_POD_LOCAL_PATH environment variable to be set to specify the location of the pod.
+if let pod_archive_path = ProcessInfo.processInfo.environment["ORT_POD_LOCAL_PATH"] {
+    // ORT_POD_LOCAL_PATH MUST be a path that is relative to Package.swift.
     //
-    // To build locally, tools/ci_build/github/apple/build_and_assemble_ios_pods.py can be used
+    // To build locally, tools/ci_build/github/apple/build_and_assemble_apple_pods.py can be used
     // See https://onnxruntime.ai/docs/build/custom.html#ios
     //  Example command:
-    //    python3 tools/ci_build/github/apple/build_and_assemble_ios_pods.py \
+    //    python tools/ci_build/github/apple/build_and_assemble_apple_pods.py \
```

Comment:
At least on the mac mini machines there's a `python3` but not `python`. Not sure if that's installed or part of the OS, but python3 does appear to be more typical on macOS.